astropy/coordinates/angle_formats.py

```diff
@@ -414,8 +414,11 @@ def degrees_to_dms(d):
 
 
 @deprecated(
-    "dms_to_degrees (or creating an Angle with a tuple) has ambiguous "
-    "behavior when the degree value is 0",
+    since="5.1",
+    message=(
+        "dms_to_degrees (or creating an Angle with a tuple) has ambiguous "
+        "behavior when the degree value is 0, {alternative}."
+    ),
```

Comment:
This is what happens in current `main` (5ac136ec0889b2829a7fb7e2065717c1b714a474):

```python
>>> from astropy.coordinates.angle_formats import dms_to_degrees
>>> dms_to_degrees(0, 0)
WARNING: AstropyDeprecationWarning: The dms_to_degrees function is deprecated and may be
removed in a future version.
        Use another way of creating angles instead (e.g. a less ambiguous string like
'-0d1m2.3s' instead. [warnings]
```

That's not good because what was supposed to be `message` does not appear at all. This is what happens in this pull request (e75032c7bb617c48353a5ed732a7b7c736828cdc):
```python
>>> from astropy.coordinates.angle_formats import dms_to_degrees
>>> dms_to_degrees(0, 0)
WARNING: AstropyDeprecationWarning: dms_to_degrees (or creating an Angle with a tuple)
has ambiguous behavior when the degree value is 0, another way of creating angles
instead (e.g. a less ambiguous string like '-0d1m2.3s'. [warnings]
```
That's better because `message` now replaces the default text, but `alternative` is not joined with `message` properly. In order to get the full message displayed properly the following is needed:
```suggestion
    message=(
        "dms_to_degrees (or creating an Angle with a tuple) has ambiguous "
        "behavior when the degree value is 0. Use {alternative}."
    ),
```
The other changes should be updated analogously.

Reply:
Good catch! I have fixed it as you suggested. Also added back a missing parenthesis to `alternative` text.